practice-one/pages/index.html

```diff
@@ -35,7 +35,7 @@
               <a href="#" class="nav-item-link">Contact Us</a>
             </li>
           </ul>
-          <button class="btn-secondary">Let's Talk</button>
+          <button class="btn btn-secondary">Let's Talk</button>
```

Comment:
Why did you naming like this for the button Let's Talk? Can you explain more about this to me?

Reply:
the class btn is a frame for button. And I use btn-secondary because I want to make it have a different color with the button Read Now